From c28ac6b4fbdbae6c91d41feb2808901ec8621ac0 Mon Sep 17 00:00:00 2001
From: Peter Pentchev <roam@ringlet.net>
Date: Fri, 4 Jan 2019 10:05:25 +0200
Subject: [PATCH] DEBIAN: Use a flag, not an ioctl value, for
 HAVE_SENDBUF_IOCTL

Forwarded: not-yet
Last-Update: 2016-04-11

The preprocessor throws a hissy fit if we try to check for
the truthiness of an ioctl definition itself, at least on PowerPC.

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 libscompat.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libscompat.m4 b/libscompat.m4
index 2bd7f5f..7b567b5 100644
--- a/libscompat.m4
+++ b/libscompat.m4
@@ -384,7 +384,7 @@ main(void)
 
 	return 0;
 }], [AC_MSG_RESULT(yes)
-     AC_DEFINE(HAVE_SENDBUF_IOCTL, TIOCOUTQ, [send buffer data])
+     AC_DEFINE(HAVE_SENDBUF_IOCTL, 1, [send buffer data])
      AC_DEFINE(SENDBUF_IOCTLVAL, TIOCOUTQ, [send buffer ioctl])],
     [AC_MSG_RESULT(no)])],
     [dnl assume no when cross-compiling
-- 
2.52.0

